src/libsync/syncengine.cpp

```diff
@@ -99,6 +99,7 @@ SyncEngine::SyncEngine(AccountPtr account, const QString& localPath,
 
 SyncEngine::~SyncEngine()
 {
+    delete _excludedFiles;
```

Comment:
Thanks for the fix!

I'd prefer to keep the smart pointer, you can just call `_excludedFiles.reset();` to achieve the same.